### PR TITLE
SW-5480: Disable submit button when uploading attachments are pending

### PR DESCRIPTION
**Pull request by tommylau523**

### CODE REVIEW

**Comment by tommylau523** [comment]
* **#2687** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2687?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @tommylau523 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: docker-compose.yml [hunk 1]
```diff
@@ -22,6 +22,11 @@ services:
       - 'SPRING_SECURITY_OAUTH2_CLIENT_REGISTRATION_KEYCLOAK_CLIENTSECRET'
       - 'SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_ISSUERURI'
       - 'SPRING_THYMELEAF_PREFIX=classpath:templates'
+      - 'TERRAWARE_ATLASSIAN_ACCOUNT'
+      - 'TERRAWARE_ATLASSIAN_APIHOST'
+      - 'TERRAWARE_ATLASSIAN_APITOKEN'
+      - 'TERRAWARE_ATLASSIAN_ENABLED'
+      - 'TERRAWARE_ATLASSIAN_SERVICEDESKKEY'
```
Comment (joshbuchea) [inline docker-compose.yml:29]: Are these changes necessary?

File: src/scenes/ContactUsRouter/ContactUsForm.tsx [hunk 2]
```diff
@@ -145,16 +150,23 @@ const ContactUsForm = () => {
...
   const supportRequestInstructions = useMemo(
     () => (supportRequestType ? getSupportRequestInstructions(supportRequestType) : ''),
-    [supportRequestType]
+    [activeLocale, supportRequestType]
+  );
+
+  // Confirming that no uploads are pending.
+  const uploadCompleted = useMemo(
+    () =>
+      allAttachments.map((attachment) => attachment.status !== 'pending').reduce((prev, curr) => prev && curr, true),
```
Comment (joshbuchea) [inline src/scenes/ContactUsRouter/ContactUsForm.tsx:168]: This can be simplified by using `.every`: `allAttachments.every((attachment) => attachment.status !== 'pending')`

**Review by joshbuchea — APPROVED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (tommylau523) [reply docker-compose.yml:29]: We need these for the BE support endpoints to work. I tested them so even when the env are not set, they will work correctly. I suspect we should add in the Dropbox or GDrive stubs here too.

File: src/scenes/ContactUsRouter/ContactUsForm.tsx (hunk 2 above)
Comment (tommylau523) [reply src/scenes/ContactUsRouter/ContactUsForm.tsx]: Thanks! Much cleaner!!
